<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(108, 99, 176);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Clientes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pesquisar_produto">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pesquisar_btn">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 131, 255);
color: rgb(255, 255, 255);
padding: 10px;
</string>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="listar_clientes_btn">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 131, 255);
color: rgb(255, 255, 255);
padding: 10px;
</string>
       </property>
       <property name="text">
        <string>Listar Clientes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cadastrar_clientes_btn">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 131, 255);
color: rgb(255, 255, 255);
padding: 10px;
</string>
       </property>
       <property name="text">
        <string>Cadastrar Clientes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="painel_clientes">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
